### ISSUE TITLE
Need Accessible Default Idle Background for Social Sharing Icons

### ISSUE THREAD

**Issue by bstifle** [Esri/calcite-web#823]
Current Background #b2b2b2 is not AA Large accessible for icons.

@paulcpederson 
Please change to $transparent-off-black:  rgba(0, 0, 0, 0.45) !default;

Reference link http://esri.github.io/calcite-web/documentation/icons/

**Comment by bstifle** [comment]
![sharing icons](https://user-images.githubusercontent.com/14083530/27561937-0ca55c34-5a7f-11e7-9cf0-252b9c3071d0.png)

**Comment by paulcpederson** [comment]
:+1:
